### PR TITLE
[CIS-734] Add Right-to-Left Support for Snapshot Tests

### PR DESCRIPTION
**Pull request by nuno-vieira**
## Description of the pull request
- Adds RTL support for snapshot tests.
- Fixes all broken SwiftUI tests that were recorded with empty size.

### CODE REVIEW

File: Sources/StreamChatUI/ChatChannelList/ChatChannelUnreadCountView+SwiftUI_Tests.swift [hunk 1]
```diff
@@ -32,6 +31,6 @@ class ChatChannelUnreadCountView_SwiftUI_Tests: XCTestCase {
 
         let view = ChatChannelUnreadCountView.SwiftUIWrapper<CustomUnreadCountView>()
         view.content = .mock(messages: 20)
-        AssertSnapshot(view)
+        AssertSnapshot(view, size: .init(width: 50, height: 50))
```
Comment (DominikBucher12) [inline Sources/StreamChatUI/ChatChannelList/ChatChannelUnreadCountView+SwiftUI_Tests.swift:34]: Why the explicit size suddenly?

Comment (nuno-vieira) [reply Sources/StreamChatUI/ChatChannelList/ChatChannelUnreadCountView+SwiftUI_Tests.swift]: @DominikBucher12 Yes you are actually right, just fixed the issue ✅

**Review by evsaev — APPROVED** [review]
🚢

**Review by DominikBucher12 — APPROVED** [review]
<3

**Pull request merged**